src/platforms/php/common/performance.mdx

```diff
@@ -0,0 +1,113 @@
+```
+
+### Instrument http calls with GuzzleHttp
+
+You need to add a [handler](http://docs.guzzlephp.org/en/stable/handlers-and-middleware.html#handlers) to your Guzzle Client. We provide one within the SDK which can be used like this:
+
+```php
+use Sentry\Tracing\GuzzleTracingMiddleware;
+...
+
+$stack = HandlerStack::create();
+
+// This is the important line
+$stack->push(GuzzleTracingMiddleware::trace());
+// ----
+
+$client = new Client([
+    'base_uri' => 'http://httpbin.org',
+    'timeout'  => 2.0,
+    'handler' => $stack
+]);
+$client->request('GET', '/get');
+```
+
+After adding this handler, you will receive spans for every request done with this client.
+
+### Manual Instrumentation
+
+To manually instrument certain regions of your code, you can create a transaction to capture them.
+
+```php
+$transactionContext = new TransactionContext();
+$transactionContext->setName('External Call');
+$transactionContext->setOp('http.caller');
+$transaction = \Sentry\startTransaction($transactionContext);
+
+$spanContext = new SpanContext();
+$spanContext->setOp('functionX');
+$span1 = $transaction->startChild($spanContext);
+
+// Calling functionX
+$this->functionX();
+$span1->finish();
+
+$transaction->finish();
+```
+
+### Retrieving a Transaction
+
+In cases where you want to attach Spans to an already ongoing Transaction you can use `SentrySdk::getCurrentHub()->getTransaction()`. This function will return a `Transaction` in case there is a running Transaction on the scope, otherwise it returns `null`.
+
+```php
+$transaction = SentrySdk::getCurrentHub()->getTransaction();
+
+if ($transaction instanceof Transaction) {
+    $transaction->setName($routeName);
+    $transaction->setData([
+        'action' => $route->getActionName(),
+        'name' => $route->getName()
+    ]);
+}
+```
+
+This can also be used if you want to filter for specifc conditions to not send a `Transaction` here is another example how to do it:
+
+```php
+$transaction = SentrySdk::getCurrentHub()->getTransaction();
+            
+if ($transaction instanceof Transaction) {
+    // $transaction->setSampled(false); // __DONT__ SEND TRANSACTION
+    // $transaction->setSampled(true); // __DO__ SEND TRANSACTION  
+}
+```
+
+### Using Relay as an Agent to improve your response time
+
+Using the performance capabilities of PHP has some impact on your response time (depending on the `traces_sample_rate` you configured).
+Because of the nature of PHP, requests in most cases, are sent in the same process as you serve your users response.
+Simply explained this means that the time it takes to send to request to Sentry, this time will be added on top of your servers response time. To mitigate that and almost reduce this time to zero, you can use a Relay running locally on the same machine or local network that acts as a Proxy / Agent. So instead of your PHP process sending things to Sentry, the PHP process sends it to your local Relay which forwards it to Sentry.
```

Comment:
```suggestion
In sum, this process affects the time it takes to send a request to Sentry, as it is added on top of your servers' response time. To mitigate this and reduce the addition close to zero, you can use a Relay running locally on the same machine or local network that acts as a proxy/agent. As a result, instead of your PHP process sending things to Sentry, the PHP process sends it to your local Relay, which then forwards it to Sentry.
```